### PR TITLE
Tableqa corrections with bumped version

### PR DESCRIPTION
**Pull request by jdpsen**
# PrimeQA Pull Request

## What does this PR do?
Changes the MRC ReadMe to have functionality brief for TableQA. Fixes a broken link in the notebook and bumps up the version.
Closes #(issue)
https://github.com/primeqa/primeqa/issues/308

## Description

Changes the MRC ReadMe to have functionality brief for TableQA. Fixes a broken link in the notebook and bumps up the version.

### Request Review

Be sure to **request a review** from one or more reviewers (unless the PR is to an unprotected branch).

## Versioning

 - [Y] Have you updated the VERSION?
 - [ ] Or does this PR not change the `primeqa` package or was not into master?

### After Merging
If merging into master and VERSION was updated, after this PR is merged:
- [ ] [Create a release](https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release) from the master with version equal to [VERSION](https://github.com/primeqa/primeqa/blob/main/VERSION)
- [ ] Not merging to master or VERSION not updated

## Checklist

Review the following and mark as completed:
- [Y ] [Tag an issue or issues](#what-does-this-pr-do) this PR addresses.
- [ Y] Added [description](#description) of changes proposed.
- [ ] Review requested as [appropriate](#request-review).
- [Y ] Version bumped as [appropriate](#Versioning).
- [ ] New classes, methods, and functions documented.
- [ ] Documentation for modified code is updated.
- [ ] Built documentation to confirm it renders as expected (see https://github.com/primeqa/primeqa/blob/main/docs/development.md).
- [ ] Code cleaned up and commented out code removed.
- [ ] Tests added to ensure all functionalities tested at >= 60% unit test coverage (see https://github.com/primeqa/primeqa/blob/main/docs/development.md).
- [ ] Code cleaned up and commented out code removed.
- [ ] Release created [as needed](#after-merging) after merging.

### CODE REVIEW

**Review by bhavani105 — APPROVED** [review]
LGTM

Remove the duplicate statement in the TableQA section
`Before continuing below make sure you have PrimeQA '
Maybe list the supported datasets along with the other MRC datasets on the top

**Review by vishwajeetkumar93 — APPROVED** [review]
LGTM

File: primeqa/mrc/README.md [hunk 1]
```diff
@@ -236,6 +236,23 @@ xlm-roberta-large -> NQ Lists: Minimal F1 = 46.95
...
+For example, if one was implementing a new model which made predictions by means other than
+an extractive head then a `NewPostProcessor` which derived predictions from the model
+outputs would be needed.
+
+Similarly, when adding support for a new dataset (with a new schema) a new preprocessor would be needed.
+This would be selected by specifying `--preprocessor qualified.path.to.new.postprocessor.NewPreProcessor`
+for the `NewPreProcessor` corresponding to this dataset and schema.
+
+
 ### PrimeQA also supports answering questions over tables
```
Comment (mihaela-bornea) [inline primeqa/mrc/README.md:256]: Moving the task arguments above the table qa section does not look right to me. TableQA is a special form of MRC and should be together with the boolean and list qa. Task arguments section should be the last one, and perhaps it needs more work. Might need to be expanded or the name should be changed, but this is outside this PR.

**Review by mihaela-bornea — COMMENTED** [review]
Please see the comment regarding the structure of the MRC README

**Review by seirasto — APPROVED** [review]
LGTM!

File: primeqa/mrc/README.md (hunk 1 above)
Comment (jdpsen) [reply primeqa/mrc/README.md]: @mihaela-bornea : We re-organized the content based on @seirasto's suggestion.

**Pull request merged**